web/wp-content/themes/alps-v3/resources/piklist/parts/widgets/author-box.php

```diff
@@ -0,0 +1,69 @@
+<?php
+  /*
+    Title: Author Box
+    Description: * This block will display a block of the post author's profile information.
+  */
+?>
+
+<div class="c-block c-block__text has-image u-theme--border-color--darker u-border--left c-block__text-expand u-spacing u-background-color--gray--light u-padding u-clear-fix can-be--dark-dark">
+
+    <?php
+        if(function_exists('get_avatar_url')) {
```

Comment:
Typically for templates, we don't use the `if (thing) {}` syntax. We use `if (thing): // code endif;` See below in this file for examples.